README.md

```diff
@@ -4,10 +4,23 @@
 ## ⚠️ Under Construction  ⚠️
 We are currently working hard to be ready for a stable release.
 
-  - Please *don't share* this repo until we make our announcement!
-  - Please try out our rc components and tell us what you think!
+  - Please *don't share* this repo until we make our public announcement!
+  - Please try out our RC components and [tell us](https://github.com/adobe-private/react-spectrum-v3/issues/new/choose) what you think!
 
-If you have feedback or would like to join our beta program, please create a [feedback issue](https://github.com/adobe-private/react-spectrum-v3/issues/new?template=Feedback.md) and let us know.
+## Overview ##
```

Comment:
Do we want to add this already or in May for the release?

Reply:
are you referring to the Overview section? For beta users directed to the GitHub, it'd be useful to have a refresher of what the project is about